test/test_tensor_ops.py

```diff
@@ -0,0 +1,411 @@
+import functools
+import torch
+
+from torch.testing._internal.common_utils import (
+    TestCase, run_tests, make_tensor, suppress_warnings)
+
+from torch.testing._internal.common_device_type import (
+    instantiate_device_type_tests, onlyOnCPUAndCUDA, ops)
+
+from torch.testing._internal.common_methods_invocations import (
+    OpInfo, SampleInput)
+
+
+class TensorCreationOpInfo(OpInfo):
```

Comment:
All OpInfo subclasses are currently defined in common_methods_invocations.py. We might start splitting that file up, but let's keep them in one place for now.